src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import core.basesyntax.OpationStrategy.OperationHandler;
+import core.basesyntax.OpationStrategy.OperationStrategy;
+import core.basesyntax.OpationStrategy.OperationStrategyImpl;
+import core.basesyntax.Operations.*;
+import core.basesyntax.ShopServiceAndReportGenerator.ReportGenerator;
+import core.basesyntax.ShopServiceAndReportGenerator.ReportGeneratorImpl;
+import core.basesyntax.ShopServiceAndReportGenerator.ShopService;
+import core.basesyntax.ShopServiceAndReportGenerator.ShopServiceImpl;
+import core.basesyntax.WorkWithFile.DataConverterImpl;
+import core.basesyntax.WorkWithFile.FileReaderImpl;
+import core.basesyntax.WorkWithFile.FileWriterImpl;
+import core.basesyntax.WorkWithFileInterface.DataConverter;
+import core.basesyntax.WorkWithFileInterface.FileDataReader;
+import core.basesyntax.WorkWithFileInterface.FileWriter;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        FileDataReader fileReader = new FileReaderImpl();
```

Comment:
The file name 'IDontKnowFileName.csv' is hardcoded. Consider parameterizing the file name to make the application more flexible and avoid potential errors if the file name changes.